src/cascadia/TerminalSettingsModel/defaults.json

```diff
@@ -335,7 +335,11 @@
 
         // Pane Management
         { "command": "closePane", "keys": "ctrl+shift+w" },
+        { "command": { "action": "splitPane", "split": "up" } },

```

Comment:
May want to stick these under `"SplitPaneParentCommandName"` instead, so they show up in the nested split pane commands

Reply:
Instead or in addition to?

Reply:
I replaced the current split horizontal/vertical with `down`/`right`, and added those as options to the extra menu below.